### PR TITLE
tidy-viewer 1.4.6

### PR DESCRIPTION
**Pull request by alexhallam**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/tidy-viewer.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class TidyViewer < Formula
   desc "CLI csv pretty printer"
   homepage "https://github.com/alexhallam/tv"
-  url "https://github.com/alexhallam/tv/archive/refs/tags/1.4.5.tar.gz"
-  sha256 "06906324bc510698651e57b1dfe9a28301ccbdd509f079bf09ce7f6d2f2fad2b"
+  url "https://github.com/alexhallam/tv/archive/refs/tags/1.4.6.tar.gz"
+  sha256 "d1c0a25d66a1acd74743db4b456c6932b8c15b586ce127a741ee74fa6455d1e9"
```
Comment (carlocab) [inline Formula/tidy-viewer.rb:5]:
```suggestion
  sha256 "e9a2fc904f2e115c715df80421c39e0f226b6750a56db96d994acfe9336ec219"
```

**Review by carlocab — COMMENTED** [review]
Not sure why the checksum doesn't match, but here's the right one.

**Comment by alexhallam** [comment]
@carlocab I made the mistake of running `sha` on my apple binary as opposed to the tar.gz binary. It is all good now.

**Review by carlocab — APPROVED** [review]
Thanks! For future reference, our preferred commit message style for version bumps is something like
```
tidy-viewer 1.4.6
```
Additionally, please don't push merge commits to your PR branch. These it more difficult to merge your PR because of our repo's linear history requirement. (That's where the `automerge-skip` label came from.)

Finally, if you have a local installation of `brew` handy, you can automate all this with something like
```sh
brew bump-formula-pr --version=1.4.6 tidy-viewer
```
Or, failing that, you can use [this GitHub Action](https://github.com/dawidd6/action-homebrew-bump-formula) to automate the creation of version bump PRs when you cut a new release.

**Comment by alexhallam** [comment]
That works.

is the command `brew bump-formula-pr --version=1.4.6 tidy-viewer` run in by local homebrew forked directory or can I run it from anywhere?

**Comment by carlocab** [comment]
You can run it anywhere. Note that `bump-formula-pr` will automatically create a new PR branch and push it to a fork on GitHub, so it will need access to credentials (but these can be cached).

**Pull request closed without merging**